src/coreclr/jit/compiler.h

```diff
@@ -10002,6 +10002,11 @@ class Compiler
     //
     bool canUseApxEncoding() const
     {
+        if (JitConfig.EnableAPX() == 0)
```

Comment:
This is not the correct way to implement a config switch for the non-altjit case.

It should be done here instead - similar how other existing instruction sets are handled:

https://github.com/dotnet/runtime/blob/6c73c19a954098d73395d310fbde54e08f86c557/src/coreclr/vm/codeman.cpp#L1441

Reply:
Thank you. I have made the change